android/quest/src/main/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensions.kt

```diff
@@ -80,7 +80,22 @@ fun List<ActionConfig>.handleClickEvent(
               NavigationArg.RESOURCE_CONFIG to actionConfig.resourceConfig,
               NavigationArg.PARAMS to interpolatedParams.toTypedArray(),
             )
-          navController.navigate(MainNavigationScreen.Profile.route, args)
+          // Issue #2685: Nullifying navOptions safeguards against unintended popBackStack
```

Comment:
please add any needed information from the issue to the in-line and remove the issue reference

Reply:
@FikriMilano The issue 2685 in the comment will be closed by this PR I do not think we need the comment if the issue has been addressed in this PR. If that is not the case then I think you need a new issue in your comment.

Reply:
okay

Reply:
Actually, delete the comment it is not necessary.